### PR TITLE
feat: high contrast theme

### PR DESCRIPTION
**Pull request by jdev082**

### CODE REVIEW

**Review by jdev082 — COMMENTED** [review]
.

**Comment by jdev082** [comment]
@rgantzos requested changes implemented

File: features/forced-dark-mode.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    const style = document.createElement('style');
+    style.textContent = styleString;
+    document.head.append(style);
+}
+
+addStyle(`
+    * {
+        background-color: invert(100%);
+    }
+`)
```
Comment (rgantzos) [inline features/forced-dark-mode.js:18]: This part is now completely unnecessary. Here's what it looks like on the Scratch homepage:
Screenshot 2022-11-23 at 10 15 17 PM

Comment (jdev082) [reply features/forced-dark-mode.js]: @rgantzos changes implemented!

Comment (rgantzos) [reply features/forced-dark-mode.js]: Not yet, images are still inverted.

Comment (jdev082) [reply features/forced-dark-mode.js]: @rgantzos Well there isn't necessarily much I can do about that other than not using the lazy method. Or I can just make a lazy method that's compatible with the lazy method and just invert the images a second time.

Comment (rgantzos) [reply features/forced-dark-mode.js]: you could do:
```
img {
   filter: invert(1);
}
```

**Comment by jdev082** [comment]
@rgantzos Alright, images are inverted a second time so that they uninvert.

File: features/forced-dark-mode.js (hunk 1 above)
Comment (jdev082) [reply features/forced-dark-mode.js]: @rgantzos ok

Comment (jdev082) [reply features/forced-dark-mode.js]: implemented!

**Comment by jdev082** [comment]
@rgantzos I think its best to call this feature high contrast.

**Comment by rgantzos** [comment]
I'll work on a few changes just to keep some of the basic elements normal.

**Comment by callumjt** [comment]
when will this be finished?

**Comment by jdev082** [comment]
Forgot about this PR

**Comment by callumjt** [comment]
> Forgot about this PR

Lol

**Pull request closed without merging**